app/tests/shared/coreFixture.js

```diff
@@ -418,5 +418,32 @@ export default {
         },
       },
     },
+
+    // Fails "6.1.22 Multiple Definition in Revision History"
+    {
+      valid: false,
+      content: {
+        ...MINIMAL_DOC,
+        document: {
+          ...MINIMAL_DOC.document,
+          tracking: {
+            ...MINIMAL_DOC.document.tracking,
+            revision_history: [
+              {
+                date: '2021-07-20T10:00:00.000Z',
+                number: '1',
+                summary: 'Initial version.',
+              },
+              {
+                date: '2021-07-21T10:00:00.000Z',
+                number: '1',
+                summary: 'Some other changes.',
+              },
+            ],
+            version: '1',
+          },
+        },
+      },
+    },
```

Comment:
I suggest to add an additional test with semantic versioning here:
```suggestion
    },

    // Fails "6.1.22 Multiple Definition in Revision History"
    {
      valid: false,
      content: {
        ...MINIMAL_DOC,
        document: {
          ...MINIMAL_DOC.document,
          tracking: {
            ...MINIMAL_DOC.document.tracking,
            revision_history: [
              {
                date: '2021-07-20T10:00:00.000Z',
                number: '1.0.0',
                summary: 'Initial version.',
              },
              {
                date: '2021-07-21T10:00:00.000Z',
                number: '1.0.0',
                summary: 'Some other changes.',
              },
            ],
            version: '1.0.0',
          },
        },
      },
    },